.changes/weak-typing.md

```diff
@@ -0,0 +1,38 @@
+---
+"tauri": patch
+"tauri-runtime": patch
+"tauri-runtime-wry": patch
+"tauri-macros": patch
+"tauri-utils": patch
+---
+
+`Params` has been removed, along with all the associated types on it. Functions that previously accepted those
+associated types now accept strings instead. Type that used a generic parameter `Params` now use `Runtime` instead. If
+you use the `wry` feature, then types with a `Runtime` generic parameter should default to `Wry`, letting you omit the
+explicit type and let the compiler infer it instead.
+
+`tauri`:
+
+* See `Params` note
+* **TODO** should we change all the `&str` to `AsRef<str>`?
+* **TODO** (probably list all the methods/functions that changed)
```

Comment:
There are too many functions and methods. I could just state the generics of many functions are updated but shouldn't affect majority of users.